plugins/com.act21.mamba.layout.alg.tree/src/com/act21/mamba/layout/alg/tree/Tree.melk

```diff
@@ -0,0 +1,37 @@
+package com.act21.mamba.layout.alg.tree
+
+import com.act21.mamba.layout.alg.tree.TreeLayoutProvider
+import org.eclipse.elk.core.math.ElkPadding
+
+bundle { 
+    metadataClass TreeMetadataProvider
+    idPrefix com.act21.mamba.layout.alg.tree
+}
+option reverseInput : boolean {
+    label "Reverse Input"
+    description
+        "True if nodes should be placed in reverse order of their
+        appearance in the graph."
+    default = false
+    targets parents
+}
+
+option numColumns : int {
+    label "Num Columns"
+    description
+        "Number of columns."
+    default = 1
+    targets parents 
+}
+
+algorithm Tree(TreeLayoutProvider) {
```

Comment:
Usually, the algorithm is specified at the top and the algorithms below. I suggest to do this here too.